active-record/README.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Active Record
+category: Architectural  
+language: en
+tags:
+ - Data access
+---
+
+## Intent
+The active record design pattern is a way of accessing data from databases by creating an object which contains the contents of a database row.
+
+## Explanation
+The ActiveRecord object is intitialised by an ActiveDatabase object which creates a connection to an exisitng database.

```

Comment:
This explanation is not sufficient. Please see other examples in the repo for what is to be expected.